src/Microsoft.Extensions.CodeGeneration.EntityFramework/Templates/DbContext/NewLocalDbContext.cshtml

```diff
@@ -23,23 +23,9 @@ using Microsoft.Data.Entity;
     } 
 }    public class @Model.DbContextTypeName : @baseClassName
     {
-        private static bool _created = false;
-
         public @(Model.DbContextTypeName)()
         {
-            if (!_created)
-            {
-                _created = true;
-                Database.EnsureCreated();
-            }
-        }
-
-        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
-        {
-        }
-
-        protected override void OnModelCreating(ModelBuilder builder)
-        {
+            
```

Comment:
No real need to define this constructor either since it's the same as what you get by default.